modules/ROOT/pages/constraints/examples.adoc

```diff
@@ -1089,6 +1092,525 @@ RETURN wrote
 ----
 
 
+[role=enterprise-edition]
+[[constraints-examples-node-property-type]]
+== Node property type constraints
+
+_This feature was introduced in Neo4j 5.9._
+
+A node property type constraint ensures that for all nodes with a certain label and property, the property is of the required property type.
```

Comment:
Initially from Jens:
This suggestion may be overly simple, but I think it is a little bit easier to follow.
```suggestion
A node property type constraint ensures that a property belonging to certain nodes has the required property type.
```

And my answer:
about the rephrasing: I think this follows the pattern we have in other sections so if we change it here maybe we should also update it for the other constraints